docs/misc/FAQ.md

```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
 
 ## Chain Data
 
-### How much ALPH are in circulation?
+### **How much ALPH are in circulation?[​](https://docs.alephium.org/frequently-asked-questions#how-much-alph-are-in-circulation)**
```

Comment:
I don't think adding bold in a header does anything. Also, this is not how you set links. You probably want something like this:

```suggestion
### [How much ALPH are in circulation?](https://docs.alephium.org/frequently-asked-questions#how-much-alph-are-in-circulation)
```

Reply:
removed all bold from the headers and fixed the links